pkg/cloud_provider/file/file.go

```diff
@@ -487,13 +498,13 @@ func (manager *gcfsServiceManager) waitForOp(ctx context.Context, op *filev1beta
 		if err != nil {
 			return false, err
 		}
-		return isOpDone(pollOp)
+		return manager.IsOpDone(pollOp)
 	})
 }
 
-func isOpDone(op *filev1beta1.Operation) (bool, error) {
+func (manager *gcfsServiceManager) IsOpDone(op *filev1beta1.Operation) (bool, error) {
 	if op == nil {
-		return false, nil
+		return true, nil
```

Comment:
in `waitForOp` above, if the `operationsService.Get` call returns nil it'll return false and retry later, now this change will break that behavior iiuc?

Reply:
I did think about it. I think if the op is nil, we should return success. This may become more relevant in multishare case, where we may have an op in cache, which has been in the cache for a long time say O(days), and eventually got garbage collected in filestore side. GET would return 404. we need to treat that op as complete and clear cache.

Eventually I plan to unify the behavior and use a single waitforop. For now I didnt want to change the single share instance code path.

I will verify with filestore team, on this behavior.

Reply:
In order to not change the non-multishare behavior should we create another `IsOpDone` instead of converting the original `isOpDone` which is called by `waitForOp` in line 495

Reply:
Ack

Reply:
Done